quickjs/common/test/app/cash/quickjs/QuickJsTest.java

```diff
@@ -85,9 +84,8 @@
     }
   }
 
-  @Ignore("https://github.com/cashapp/quickjs-java/issues/214")
   @Test public void emojiRoundTrip() {
-    assertThat(quickjs.evaluate("\"\uD83D\uDC1D️\""))
-        .isEqualTo("\uD83D\uDC1D️️");
+    assertThat(quickjs.evaluate("\"a\uD83D\uDC1Dcdefg\""))
+        .isEqualTo("a\uD83D\uDC1Dcdefg");
```

Comment:
on trunk this fails in two ways. The emoji is encoded wrong, and the remainder of the string is truncated.

```
expected: a🐝cdefg
but was : aðcd
```